docs/sql-ref-syntax-aux-describe-table.md

```diff
@@ -18,5 +18,53 @@ license: |
   See the License for the specific language governing permissions and
   limitations under the License.
 ---
+### Description
+DESCRIBE TABLE statement is used to return the metadata information of a
+table. The basic metadata information includes column name, column type
+and column comment. If EXTENDED mode is specified then additional
+metadata such as parent database, owner, created and last access time etc.
+are returned. Optionally a partition spec or column name can be specified
+to return the metadata pertaining to the partition or column respectively.
 
-**This page is under construction**
+### Syntax
+{% highlight sql %}
+{DESC | DESCRIBE} [TABLE] [format] tableIdentifier [partition-spec] [desc-col]
+
+format:= EXTENDED | FORMATTED
+tableIdentifier:= [db.][table]
+parition-spec:= PARTITION (part_col1 = value1, part_col2 = value2, ...) 
+desc-col:= [db.][table.]column-name
+{% endhighlight %}
+**Note**<br>
```

Comment:
Do you need `<br>`? try adding blank lines instead

Reply:
@srowen No.. did not need it. Not sure why i put it in the first place. Thanks for catching it.